src/recordlinker/assets/initial_algorithms.json

```diff
@@ -25,7 +25,12 @@
                         "func": "func:recordlinker.linking.matchers.compare_probabilistic_fuzzy_match"
                     }
                 ],
-                "rule": "func:recordlinker.linking.matchers.rule_probabilistic_match",
+                "rule": "func:recordlinker.linking.matchers.rule_probabilistic_sum",
+                "possible_match_window": [
+                    0.8,
+                    0.925
+                ],
+                "maximum_points": 13.2001963043,
```

Comment:
Does this need to be specified?  We know the log odds values and we know the evaluators in use for a particular pass, it seems like the algorithm should just be able to calculate this.

Reply:
Great catch, not in the slightest do we need to define this here. I just started in this file and then changed some structure later, so I can go back and define this calculation on the fly.